src/EnergyPlus/VariableSpeedCoils.cc

```diff
@@ -5834,6 +6069,124 @@ namespace VariableSpeedCoils {
 
 	}
 
+	void SetAirNodes(std::string const & CoilName, // must match coil names for the coil type
```

Comment:
Recommended cleanups for this function.  Change declaration style (again, until we have a better style defined) to:

```
void SetAirNodes(
        std::string const & CoilName,
        etc...
) {
        code
}
```

Remove all the unnecessary header blocks ( REFERENCES, INTERFACE BLOCK SPECIFICATIONS, etc. )

Remove commented out code that isn't useful: 
`//    WaterIndex=FindGlycol('WATER') !Initialize the WaterIndex once`